crest/Assets/Crest/Crest/Shaders/Ocean.shader

```diff
@@ -171,6 +171,8 @@ Shader "Crest/Ocean"
 		[Header(Clip Surface)]
 		// Discards ocean surface pixels. Requires 'Create Clip Surface Data' enabled on OceanRenderer script.
 		[Toggle] _ClipSurface("Enable", Float) = 0
+		// Clips purely based on water depth
+		[Toggle] _ClipUnderTerrain("Clip under terrain", Float) = 0
```

Comment:
Using "Terrain" might cause confusion since it will happen to anything that adds depth. Some might take it as Unity Terrain too. "ClipUnderDepthCacheObjects" or something doesn't sound as accessible though.

Reply:
Yeah fair point! How about "clip below land" perhaps?

Reply:
We can probably keep terrain with something like "Clip Under Terrain (requires depth cache)"

Reply:
Ok can go for that